pkg/gitops/gitops.go

```diff
@@ -65,7 +65,7 @@ func RunLoop(url, branch string) error {
 				})
 			case gitops.UpdateTypeDeleted:
 				go runHandle(wg, func() error {
-					return handleDelete(file.APIType)
+					return handleDelete(vm)
```

Comment:
this will panic, as VM is not defined here; it is `nil` as it has been removed.
instead, construct a new VM object which you pass, based on the API type

Reply:
Fixed in ad0cc62.